apps/design/backend/src/test_decks.test.ts

```diff
@@ -104,7 +108,9 @@ describe('createPrecinctTestDeck', () => {
       precinctId,
       ballots,
     });
-    await expect(testDeckDocument).toMatchPdfSnapshot();
+    await expect(testDeckDocument).toMatchPdfSnapshot({
+      failureThreshold: PDF_SNAPSHOT_DIFF_THRESHOLD,
+    });
```

Comment:
Had to add some wiggle room here as was already the case for the report snapshot comparison below. Will post the diff images in a moment to confirm that nothing of substance changed (I'm a bit surprised that anything changed at all)